install/kubernetes/helm/istio/templates/operator-configmap.yaml

```diff
@@ -0,0 +1,1241 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "istio.name" . }}
+    chart: {{ template "istio.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  config: |-
```

Comment:
Can you read this from a file rather than inlining?

Reply:
+1

Reply:
One way would be to include 2 files - one the original crd yaml (from 1.0), and one crd-1.1.yaml. I believe we need to keep the 1.0 yaml around as is, and it might be a good practice to have separate files for each version.

Reply:
agreed re separate YAMLs for each version - this was in my original PR on this topic.  I was moving ahead a little too fast to reintegrate them.

Reply:
in the galley PR, this is read from files in helm/files directory.